core/auth/local/user_management.py

```diff
@@ -20,7 +20,8 @@
     "oneshotanalytics": {"read": True, "write": True, "toggle": True, "run": True},
     "tag": {"read": True, "write": True},
     "export": {"read": True, "write": True, "toggle": True, "refresh": True},
-    "files": {"read": True, "write": True},
+    "attachedfiles": {"read": True, "write": True},
+    "file": {"read": True, "write": True},
```

Comment:
Migrations in a schema-less database? Sigh. Maybe a script that takes the schema version into account and applies changes would be enough. Let me see if I can come up with a quick way to do this.